msticpy/data/query_source.py

```diff
@@ -94,6 +94,8 @@ def __init__(
         self.metadata = ChainMap(
             _value_or_default(self._source, "metadata", {}),
             _value_or_default(self.defaults, "metadata", {}),
+            # self._source.get("metadata", {}),
```

Comment:
These have sneaked back in

Reply:
Good spot, must have come back in one of the merges.